src/opensignals/data/provider.py

```diff
@@ -208,6 +208,11 @@ def download_data(self, db_dir: pathlib.Path, recreate: bool = False) -> None:
 
         ticker_data = self.get_ticker_data(db_dir)
         ticker_map = self.get_tickers()
+
+        ticker_map[['_', 'region']] = ticker_map['bloomberg_ticker'].str.split(' ', n=2, expand=True)
```

Comment:
I would rather could override / decorate the get_tickers method to do this mapping. In that way the implementation specific knowledge stays within the IEXCloud provider. The region parameter could be a constructor parameter of IEXCloud provider.